integrations/sql/src/main/kotlin/com/xebia/functional/xef/sql/SQL.kt

```diff
@@ -15,9 +15,13 @@ import kotlinx.serialization.json.Json
 interface SQL {
 
   companion object {
-    suspend fun <A> fromJdbcConfig(config: JdbcConfig, block: suspend SQL.() -> A): A = JDBCSQLImpl(config).use {
+    suspend fun <A> fromJdbcConfig(config: JdbcConfig, block: suspend SQL.() -> A) : A = JDBCSQLImpl(config).use {
       block(it)
     }
+
+    fun fromJdbcConfigSync(config: JdbcConfig): SQL {
```

Comment:
Function "fromJdbcConfigSync" is never used

Reply:
This is being used in AIDatabase.java